pss/pss.go

```diff
@@ -455,17 +465,30 @@ func (p *Pss) deregister(topic *message.Topic, hndlr *handler) {
 	delete(handlers, hndlr)
 }
 
+// generic peer-specific handler for incoming messages
+// calls pss msg handler asyncronously
+func (p *Pss) handle(ctx context.Context, peer *protocols.Peer, msg interface{}) error {
+	go func() {
+		pssmsg, ok := msg.(*message.Message)
+		if !ok {
```

Comment:
this is not necessary, `p2p/protocols` package guarantees correct type, and if we added another message the handler should match on type anyway